src/prototype_room_my.js

```diff
@@ -284,7 +284,31 @@ Room.prototype.getHarvesterAmount = function() {
   return amount;
 };
 
+Room.prototype.visualizz = function() {
+
+  let color = function(coeff) {
+    return '#' + (Math.floor(255 - coeff * 239)).toString(16) + (Math.floor(coeff * 239 + 16)).toString(16) + '00';
```

Comment:
RoomVisual understands any web color, including `'rgb(256, 256, 256)'`. So no need to hex conversion)

Reply:
great, thx